server/service/client.go

```diff
@@ -1608,25 +1608,35 @@ func (c *Client) DoGitOps(
 		team["software"].(map[string]any)["app_store_apps"] = config.Software.AppStoreApps
 		team["software"].(map[string]any)["packages"] = config.Software.Packages
 		team["secrets"] = config.TeamSettings["secrets"]
-		team["webhook_settings"] = map[string]interface{}{}
-		clearHostStatusWebhook := true
-		if webhookSettings, ok := config.TeamSettings["webhook_settings"]; ok {
-			if _, ok := webhookSettings.(map[string]interface{}); ok {
-				if hostStatusWebhook, ok := webhookSettings.(map[string]interface{})["host_status_webhook"]; ok {
-					clearHostStatusWebhook = false
-					team["webhook_settings"].(map[string]interface{})["host_status_webhook"] = hostStatusWebhook
-				}
-			} else if webhookSettings != nil {
-				return nil, fmt.Errorf("team_settings.webhook_settings config is not a map but a %T", webhookSettings)
-			}
```

Comment:
Refactored this a bit to be similar to the equivalent code on the global side of things, which is a bit cleaner and more repeatable.  I don't think we support per-team vuln or activity webhooks, but if we do in the future then this pattern will be easy to follow.